pages/dkp/kommander/2.2/dkp-upgrade/upgrade-konvoy/essential/index.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: layout.pug
+navigationTitle: DKP Essential
+title: Upgrade Konvoy for DKP Essential
+excerpt: Upgrade your Konvoy environment within the DKP Essential License.
+beta: false
+enterprise: false
+menuWeight: 30
+---
+## Prerequisites
+
+* Follow the steps listed in the [DKP upgrade overview][dkpup]
+
+* Ensure that all platform applications in the management cluster have been upgraded to avoid compatibility issues with the current Kubernetes version. This is done automatically when [upgrading Kommander][upgradekomm], so ensure that Kommander has been upgraded prior to Konvoy.
```

Comment:
```suggestion
* Ensure that all platform applications in the management cluster have been upgraded to avoid compatibility issues with the [Kubernetes version][releasenotes] included in this release. This is done automatically when [upgrading Kommander][upgradekomm], so ensure that Kommander has been upgraded prior to Konvoy.
```